src/main/java/LogOut/ui/LogOutView.java

```diff
@@ -0,0 +1,30 @@
+package LogOut.ui;
+
+import LogOut.interface_adapters.LogOutController;
+
+import javax.swing.*;
+import java.awt.*;
+import java.beans.PropertyChangeEvent;
+import java.beans.PropertyChangeListener;
+
+public class LogOutView extends JPanel implements PropertyChangeListener {
+
+    private final LogOutController controller;
+
+    public LogOutView(LogOutController controller){
+        this.controller = controller;
+    }
+
+    @Override
+    public void propertyChange(PropertyChangeEvent evt) {
+        if (evt.getPropertyName().equals("Log Out")){
+            String logOut = "Log Out";
+            LogOutController logOutController = new LogOutController(logOut);
+            controller.LogOutInitializer(logOutController);
+
+
+            CardLayout cardLayout = (CardLayout) this.getLayout();
```

Comment:
Im not sure if this is the correct way to get the main page view (not sure what it's called) and then show the main login/signup view

Reply:
hmm I'm not quite sure either...

Reply:
The switch here will be done in the main frame i think, as we are switching the card with the main view to the card with the landing view. I did not cover the converse of this in my use case, as we will need to implement the main view, which is just a frame.